lib/class-wp-theme-json-gutenberg.php

```diff
@@ -379,8 +379,9 @@ class WP_Theme_JSON_Gutenberg {
 			'minHeight' => null,
 		),
 		'layout'                        => array(
-			'contentSize' => null,
-			'wideSize'    => null,
+			'contentSize'  => null,
+			'wideSize'     => null,
+			'allowEditing' => null,
```

Comment:
Doesn't need to happen in this PR, but should we add a `@since 6.4.0` line in the doc comment above this `const`?